### PR TITLE
Implement 404 and 500 Page

### PR DESCRIPTION
**Pull request by anthotsang**
https://app.asana.com/0/683768070860536/846826093023571

### CODE REVIEW

File: app/assets/javascripts/errors.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (hanchang) [inline app/assets/javascripts/errors.js]: Let's go ahead and delete

File: app/assets/stylesheets/errors.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the errors controller here.
```
Comment (hanchang) [inline app/assets/stylesheets/errors.scss]: Let's go ahead and delete this file, doesn't seem needed.

File: app/controllers/concerns/responses.rb [hunk 3]
```diff
@@ -6,6 +6,7 @@ module Responses
   end
 
   def render_404
+    puts "render 404"
```
Comment (hanchang) [inline app/controllers/concerns/responses.rb]: Let's remove debugging statement here.

File: app/helpers/errors_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module ErrorsHelper
```
Comment (hanchang) [inline app/helpers/errors_helper.rb]: Let's go ahead and delete this file, doesn't seem needed.

File: app/views/errors/internal_server_error.html.slim [hunk 5]
```diff
@@ -1,23 +1 @@
...
-      li= link_to "Our Blog", "https://blog.coinfi.com", class: "mid-gray"
-      li= link_to "About Us", page_about_path, class: "mid-gray"
-    div.flex.items-center.mb3.mt4
-      h1.f4.fw7.mv0.lh-title Contact Us
-      = image_tag "chat_bubble_outline.png", :class => "mw-100 ml2"
-    div.f5
-      | If you still can't find what you're looking for or if this was a mistake, 
-      | shoot us an email at <a href="mailto:contact@coinfi.com">contact@coinfi.com</a>
\ No newline at end of file
+= render "errors/template", { message: "Sorry, something went wrong." }
```
Comment (anthotsang) [inline app/views/errors/internal_server_error.html.slim]: Thanks! I wasn't sure if refactoring like this would be premature or not.

**Pull request merged**